temba/msgs/models.py

```diff
@@ -1540,3 +1542,10 @@ class MessageExportAssetStore(BaseExportAssetStore):
     directory = "message_exports"
     permission = "msgs.msg_export"
     extensions = ("xlsx",)
+
+
+class MessageExternalIDMap(models.Model):
+    message = models.ForeignKey(Msg, on_delete=models.CASCADE, related_name="external_ids")
+    channel = models.ForeignKey(Channel, on_delete=models.CASCADE)
+    carrier_id = models.CharField(max_length=64, blank=True, null=True)
```

Comment:
Quick question: could this field be an integer? Or does it need to be a string?

Reply:
Also, do you think that we should have the created_at and updated_at fields?

Reply:
> @ihor-palii - Quick question: could this field be an integer? Or does it need to be a string?

In mGage docs, this id could contain characters so I that is why I made it CharField (per what I have tested I saw only numbers).

Reply:
> Also, do you think that we should have the created_at and updated_at fields?

That's, a good Idea I will add these two fields.

Reply:
> > @ihor-palii - Quick question: could this field be an integer? Or does it need to be a string?
> 
> In mGage docs, this id could contain characters so I that is why I made it CharField (per what I have tested I saw only numbers).

Got it, so let's keep CharField :)